webpack/assets/javascripts/react_app/components/SettingUpdateModal/components/SettingForm/SettingForm.js

```diff
@@ -28,6 +28,12 @@ const SettingForm = ({
       item: 'Settings',
       message: __('Setting was successfully updated.'),
       method: 'put',
+      errorAlert: error => {
+        const {
+          response: { data },
+        } = error;
+        return data.error ? data.error.full_messages : error;
+      },
```

Comment:
Maybe rename to `customErrorAlert`

Didn't test yet:
```suggestion
      customErrorAlert: ({response: {data}, message}) => data.error ? data.error.full_messages : message,
```